src/Umbraco.Cms.Api.Management/DependencyInjection/UmbracoBuilderExtensions.cs

```diff
@@ -84,6 +85,7 @@ public static IUmbracoBuilder AddUmbracoManagementApi(this IUmbracoBuilder build
 
             services.ConfigureOptions<ConfigureUmbracoBackofficeJsonOptions>();
             services.ConfigureOptions<ConfigureUmbracoManagementApiSwaggerGenOptions>();
+            builder.Services.ConfigureOptions<ManagementApiMvcConfigurationOptions>();
```

Comment:
Why not just `services` as it refers to `IServiceCollection services = builder.Services;`?

Reply:
I've aligned this now to use `builder.Services` throughout this method (didn't seem to me the `services` variable was adding much value).